test/it/authenticators-deactivate.ts

```diff
@@ -23,20 +24,21 @@ describe('Authenticators API tests', () => {
     }
   });
 
-  it('should deactivate an active Authenticator', async () => {
-    const authenticators = await client.listAuthenticators();   // returns Collection<Authenticator>
+  // OKTA-483353
+  xit('should deactivate an active Authenticator', async () => {
```

Comment:
maybe some MFA enrollment policy are not cleared after test.

Reply:
this appears to be an issue even in isolated run (running only this test for the first time after policy reset succeeds, but all subsequent runs fail)